### ISSUE TITLE
Default config file doesn't work on Linux

### ISSUE THREAD

**Issue by walbers** [fluent/fluent-bit#4635]
## Bug Report

**Describe the bug**
The fluent-bit.conf file downloaded with the 64 bit zip for Windows doesn't work on Windows.

**To Reproduce**
- Windows allows you to create the service but the service will never get out of the `STOPPED` state
- Steps to reproduce the problem:
1. `sc.exe create fluent-bit binPath= "C:\test\bin\fluent-bit.exe -c C:\test\conf\fluent-bit.conf"`
2. `sc.exe start fluent-bit`
3. `sc.exe query fluent-bit` 
4. Query output:
```
SERVICE_NAME: fluent-bit
        TYPE               : 10  WIN32_OWN_PROCESS
        STATE              : 1  STOPPED
        WIN32_EXIT_CODE    : 1067  (0x42b)
        SERVICE_EXIT_CODE  : 0  (0x0)
        CHECKPOINT         : 0x0
        WAIT_HINT          : 0x0
```

**Expected behavior**
For the service to be started. This was fixed after hours of debugging. I tried the config file found [here](https://docs.fluentbit.io/manual/pipeline/inputs/windows-event-log) and everything worked fine.

**Your Environment**

* Version used: Latest
* Configuration: default everything except moved to folder named `test`
* Operating System and version: Windows 10
* Filters and plugins: None

**Comment by agup006** [comment]
@walbers I'm changing the title to Windows, @fujimotos have you seen this before?

**Comment by fujimotos** [comment]
> The fluent-bit.conf file downloaded with the 64 bit zip for Windows doesn't work on Windows.
>>  I'm changing the title to Windows, @fujimotos have you seen this before?

@agup006 The reason is pretty simple: the deafault configuration file
`fluent-bit.conf` is meant for Unix. It's not never intended for Windows.

For reference, I posted a minimum configuration that works on Windows
to #4647.  I hope that patch clarify things for you.

**Comment by fujimotos** [comment]
>  I'm changing the title to Windows, @fujimotos have you seen this before?

@agup006 The reason is pretty simple: the default configuration file
`fluent-bit.conf` is meant for Unix. It's not never intended for Windows.

For reference, I posted a minimum configuration that works on Windows
to #4647.  I hope that patch helps.

**Comment by patrick-stephens** [comment]
@fujimotos do you want me to merge that config over to the docs https://docs.fluentbit.io/manual/installation/windows?
That'll help prevent this issue as well.

**Comment by niedbalski** [comment]
@patrick-stephens https://github.com/fluent/fluent-bit/issues/3720 intends to address the windows builds issue. As for this specific, In my opinion a simple smoke test that uses the configuration pointed by @fujimotos can help here.

**Comment by patrick-stephens** [comment]
> @patrick-stephens #3720 intends to address the windows builds issue. As for this specific, In my opinion a simple smoke test that uses the configuration pointed by @fujimotos can help here.

Yeah I'll look at that next week if I can, just add a job for smoke testing it rather than the full CI.

**Comment by patrick-stephens** [comment]
@fujimotos also did a follow up PR that makes the new config file the default which is super helpful and what I'll try to pull down for testing/verification as simplifies things: https://github.com/fluent/fluent-bit/pull/4679

**Comment by patrick-stephens** [comment]
> > @patrick-stephens #3720 intends to address the windows builds issue. As for this specific, In my opinion a simple smoke test that uses the configuration pointed by @fujimotos can help here.
> 
> Yeah I'll look at that next week if I can, just add a job for smoke testing it rather than the full CI.

I've got a working POC now I'll aim to integrate into the workflows: https://github.com/patrick-stephens/fluent-bit-release-test/runs/4954750346?check_suite_focus=true

https://github.com/patrick-stephens/fluent-bit-release-test/blob/42072c455294e1d3bf4def7f9d725171e4cca38e/.github/workflows/windows-ci.yaml
